lib/util.js

```diff
@@ -46,3 +46,7 @@ exports.seticonflag = function (path, cb) {
   buf.writeUInt8(4, 8)
   xattr.set(path, 'com.apple.FinderInfo', buf, cb)
 }
+
+exports.codesign = function (identity, path, cb) {
+  exports.sh('codesign', ['--verbose', '--force', '--sign', identity, path], function (err) { cb(err) })
```

Comment:
Why pass the `--force` flag? It's a fresh unsigned DMG before this step so I don't see what it gives us.

Reply:
I'm also wondering if the identifier should be set explicitly.

Reply:
Good point about the `--force` flag. What do you mean about explicitly setting the identifier?

Right now, this works if I set, e.g., `"code-sign-identity": "Developer ID Application: My Company, Inc."` in my spec json.

Reply:
>      -i, --identifier identifier
>            During signing, explicitly specify the unique identifier string that is embedded in code signatures. If this option is omitted,
>            the identifier is derived from either the Info.plist (if present), or the filename of the executable being signed, possibly modi-
 >            fied by the --prefix option.  It is a very bad idea to sign different programs with the same identifier.
From the `codesign` manual.

Reply:
Yeah, it's probably a good idea to include that, at least as an option. I'll update the PR.